tests/Unit/Core/Form/IdentifiableObject/DataProvider/ProductFormDataProviderTest.php

```diff
@@ -1505,4 +1618,12 @@ private function mockCategoryDataProvider(): CategoryDataProvider
 
         return $categoryDataProvider;
     }
+
+    private function mockProductImagePathFactory(): ProductImagePathFactory
```

Comment:
You won't need to mock this anymore once the image is handled in the query

Reply:
it makes sense

Reply:
pushed